pkg/apis/samples/v1alpha1/samplesource_defaults.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+)
+
+// SetDefaults mutates SmapleSource.
+func (s *SampleSource) SetDefaults(ctx context.Context) {
+	//Add code for Mutating admission webhook.
+
```

Comment:
crazy idea: what if we made the `spec.interval` variable optional, but then set a default (say, 10s?) here to demonstrate the defaulting webhook as well?

Reply:
Right! I will add it also.